buildSrc/src/main/groovy/io/micronaut/gradle/MicronautWebsitePlugin.groovy

```diff
@@ -207,6 +220,18 @@ class MicronautWebsitePlugin implements Plugin<Project> {
                 }
             }
         })
+        project.tasks.register(TASK_CLEAN_MN_LOGOS, { task ->
```

Comment:
the clean task should depend on this task. 

The idea is if you run ./gradlew clean it cleans everything

Reply:
Got it. That makes sense.